docs/core/getting_started/thinking-prefectly.md

```diff
@@ -2,13 +2,9 @@
 sidebarDepth: 0
 ---
 
-# First Steps
+# Thinking Prefectly
 
-Welcome to Prefect! These docs will give you a gentle introduction to Prefect's core concepts. 
-
-We also have a [full tutorial](/core/tutorial/01-etl-before-prefect.html) on building real-world data applications with Prefect.
-
-## Thinking Prefectly
+This page will give you a gentle introduction to Prefect's core concepts. If you are keen to get started we have a [Getting Started guide](/core/getting_started.html) and [ETL tutorial](/core/tutorial/01-etl-before-prefect.html) on building real-world data applications with Prefect.
```

Comment:
hmmm where is this linking to exactly?  This file is now part of the "Getting Started" section in Core, it feels weird to direct them to a Getting Started guide from within a Getting Started guide

Reply:
Thinking Prefectly is in an "About Prefect" section.  I've moved docs to make this clearer.